script.js

```diff
@@ -0,0 +1,68 @@
+const contactForm = document.querySelector('.book_form');
+const title = document.querySelector('.book_title');
+const author = document.querySelector('.book_author');
+const bookList = document.querySelector('.books-list');
+let titleArray = [];
+let authorArray = [];
+
+function intitializeDocument() {
+  bookList.innerHTML = '';
+  for (let i = (titleArray.length - 1); i > -1; i -= 1) {
+    bookList.innerHTML = `${bookList.innerHTML}<p>${titleArray[i]}</p>
+                            <p>${authorArray[i]}</p>
+                            <button class="remove_button_${i}">remove</button><hr/>`;
+  }
+}
+
+function removeBook(index) {
+  document.querySelector(`.remove_button_${index}`).addEventListener('click', (event) => {
+    const tempArrayTitle = [];
+    const tempArrayauthor = [];
+    for (let j = 0; j < titleArray.length; j += 1) {
+      if (j !== event.currentTarget.index) {
+        tempArrayTitle.push(titleArray[j]);
+        tempArrayauthor.push(authorArray[j]);
+      }
+    }
+    titleArray = tempArrayTitle;
+    authorArray = tempArrayauthor;
+    localStorage.setItem('title', JSON.stringify(titleArray));
+    localStorage.setItem('author', JSON.stringify(authorArray));
+    intitializeDocument();
+    /* eslint-disable */
+    intitializeRemoveButtonEvents();
+   /* eslint-enable */
+  });
+  document.querySelector(`.remove_button_${index}`).index = index;
+}
+
+function intitializeRemoveButtonEvents() {
+  for (let i = (titleArray.length - 1); i > -1; i -= 1) {
+    removeBook(i);
+  }
```

Comment:
- [ ] OPTIONAL: I recommend you to use ES6 syntax arrow function: 
`const intitializeRemoveButtonEvents = ()  => { }`
- [ ] OPTIONAL: I recommend you to use ES6 syntax to iterate through array and 
`books.forEach  {(book) => console.log(book.title }`
- [ ] OPTIONAL: I recommend you to use ES6 syntax to filter and delete book:

```
document.getElementsByID('button').forEach((btn, i) => {
    btn.addEventListener('click', () => {
      // here is the remove logic
      books = books.filter((book, k) => i !== k);
      intitializeDocument();
    });
  });
}

```

Reply:
yeah i considered using an in-built filter function from the ES6 but it was some milliseconds slower than the one i wrote custom made by my self. and some of the ES6 foreach loop runs a little slower than using the regular inbuilt language for loops. Thanks for pointing all of these out. 😉